circle.yml

```diff
@@ -50,13 +50,13 @@ dependencies:
     - docker-compose version
     - docker version
   override:
-    - .circle/docker-compose.sh pull
+    - .circle/docker-compose.sh pull ${DISTRO}
   post:
-    - .circle/docker-compose.sh build
+    - .circle/docker-compose.sh build ${DISTRO}
```

Comment:
Also what is advantage of these above? Also not clear. I do understand if the variable required in a bash singlet on a particular step. However docker-compose.sh is self sufficient with it's build env...